style.css

```diff
@@ -0,0 +1,71 @@
+body {
+  background-image: url(peace.jpg);
+  background-repeat: repeat-y;
```

Comment:
Not needed

Reply:
Ιt is necessary because the photo appears many times